### PR TITLE
Shina/ch6121/review and update sportify bigquery

### PR DESCRIPTION
**Pull request by odidere**
Story details: https://app.clubhouse.io/uncoil/story/6121/review-and-update-required-documentation

For task number 5.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,6 +42,117 @@ val df = sqlContext.bigQuerySelect(
 df.saveAsBigQueryTable("my-project:my_dataset.my_table")
 ```
 
+# Why this repo?
+
+This repository was created because the parent project `sportify/bigquery` is no longer being actively developed.
```
Comment (jakeorr) [inline README.md]: You have `portify/bigquery`. Should be `spark-bigquery`

File: README.md [hunk 2]
```diff
@@ -42,6 +42,117 @@ val df = sqlContext.bigQuerySelect(
...
+```bash
+$sbt
+>publishSigned
+```
+
+You can find more details on https://central.sonatype.org/pages/ossrh-guide.html.
+
+# Further work
+
+I was originally asked to use the package `io.github.odidere` by maven central admin, a reversal of my github address. This should normally be `com.uncoil` or `ai.unicoil`, the reverse of Uncoil.ai. Note that the company must own any domain that is chosen.
```
Comment (jakeorr) [inline README.md]: `ai.uncoil`

**Review by jakeorr — APPROVED** [review]
Looks good, just a couple typos that I noticed.

File: README.md (hunk 1 above)
Comment (odidere) [reply README.md]: @jakeorr the original project was `sportify/spark-bigquery`

**Pull request merged**